crates/egui/src/widgets/color_picker.rs

```diff
@@ -311,6 +340,49 @@ fn color_picker_hsvag_2d(ui: &mut Ui, hsva: &mut HsvaGamma, alpha: Alpha) {
     color_slider_2d(ui, v, s, |v, s| HsvaGamma { s, v, ..opaque }.into());
 }
 
+/// Shows 4 `DragValue` widgets to be used to edit the Color32's RGBA values.
+/// Alpha's `DragValue` is hidden when `Alpha::Opaque`.
+///
+/// Returns `Some(Color32)` on change.
+fn srgba_edit_ui(ui: &mut Ui, rgba: Color32, alpha: Alpha) -> Option<Color32> {
```

Comment:
Using premultiplied alpha in a UI is just confusing, especially since the rest of the color picker separates out the alpha.

Reply:
Are you referring to the DragValues? The values they display and let the user change are not premultiplied with the alpha channel, the premultiplied alpha values are only shown in the color_text_ui below the DragValues.

Reply:
The `DragValues` below takes the alpha from a `Color32`, which per definition is always pre-multiplied.

…but I just now realize the code on `master` is already displaying pre-multiplied values, which is my bad. We should never use pre-multiplied alpha in the UI imho.